api/external/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -0,0 +1,147 @@
+syntax = "proto3";
```

Comment:
Moved this proto from Gloo into solokit https://github.com/solo-io/gloo/pull/4777/files#r639104211